Modules/+Experiments/+AutoExperiment/@AutoExperiment_invisible/AutoExperiment_invisible.m

```diff
@@ -180,23 +199,7 @@ function abort(obj)
             end
             obj.logger.log('Abort requested');
         end
-        function pause(obj,~,~)
```

Comment:
Is this functionality handled somewhere else now instead?

Reply:
it was never linked up with anything.